examples/mobile/UIComponents/components/appbar/more-option-list-max-width.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+	<style>
+	</style>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<select data-native-menu="false">
+					<option data-placeholder="true" value="first-line" data-hide-placeholder-menu-items="false">
+						Open
+					</option>
+					<option value="1">
+						Option1
+					</option>
+					<option value="2" class="ui-dropdown-two-lines">
+						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam consectetur venenatis blandit. Praesent vehicula, libero non pretium vulputate, lacus arcu facilisis lectus, sed feugiat tellus nulla eu dolor. Nulla porta bibendum lectus quis euismod. Aliquam volutpat ultricies porttitor. Cras risus nisi, accumsan vel cursus ut, sollicitudin vitae dolor. Fusce scelerisque eleifend lectus in bibendum. Suspendisse lacinia egestas felis a volutpat.
```

Comment:
Can't it be changed to 2 lines?
The code width is too long.

Reply:
This is expected. User gives long text and widget formats it into two lines.